CodeStarterCamp_Week2/main.swift

```diff
@@ -6,7 +6,40 @@
 //  Copyright © yagom academy. All rights reserved.
 //
 
-import Foundation
+let myLottoNumbers: [Int] = [4, 31, 11, 7, 28, 45]
+
+var lottoNumbers: Set<Int> = Set<Int>()
+
+func lottoPlease() -> Set<Int>{
+    repeat {
+        let lottoNumber = Int.random(in: 1...45)
+        lottoNumbers.insert(lottoNumber)
+        if lottoNumbers.contains(lottoNumber) {
+        } else {
+            lottoNumbers.insert(lottoNumber)
+        }
```

Comment:
음.. lottoNumbers에 데이터가 있는지 없는지 체크하는 로직이군요!
그런데.. 이 조건문이 꼭 필요한가요? 👀
오히려 불필요한 동작을 추가하고 있다는 생각이 듭니다.

그리고 위 사항과 별개로 조건을 만족할 때 실행할 로직이 전혀 없네요! 😅
이럴 경우 조건을 반대로 걸어주면 좀 더 깔끔하게 코드를 작성할 수 있을거 같아요.
```suggestion
        if lottoNumbers.contains(lottoNumber) == false {
            lottoNumbers.insert(lottoNumber)
        }
```

Reply:
앗 제가 고민했던 부분인데 '고민했던 점'에 깜빡하고 작성하지 않았네요. 제가 하고 싶은 코드가 wonbi님께서 적어주신 그대로입니다!! 앞으로 참고해서 잘 사용하겠습니다.